hexa/workspaces/schema/types.py

```diff
@@ -96,6 +102,26 @@ def resolve_workspace_members(workspace: Workspace, info, **kwargs):
     )
 
 
+@workspace_object.field("invitations")
+def resolve_workspace_invitations(workspace: Workspace, info, **kwargs):
+    request: HttpRequest = info.context["request"]
+
+    status = [WorkspaceInvitationStatus.ACCEPTED, WorkspaceInvitationStatus.PENDING]
+    if kwargs.get("status"):
```

Comment:
I would prefer to have a second filter on the query set in case there is a condition on the status

```python
if kwargs.get("status"):
  qs = qs.filter(status=kwargs["status"])



```